spec/shop_spec.rb

```diff
@@ -27,8 +27,20 @@
   describe "#all" do
     rows = DB[:shops].all
     it 'should return all rows from the shops table' do
-      shop = Shop.new "cafe"
+      shop = Shop.new
       expect(Shop.all).to eq(rows)
     end
   end
+
+  describe "#find" do
+    shop = Shop.new({name: 'RED', description: 'red apples'})
+    v = Shop.find(15)
+    it 'should return row with id 15 from the shops table' do
```

Comment:
maybe it's better something like:`it "returns the given shop row from shops table"`